sanssouci/lambda_calibration.py

```diff
@@ -91,50 +87,49 @@ def get_permuted_p_values_one_sample(X, B=100, seed=None):
     """
     Get permutation p-values: Get a matrix of p-values under the null
     hypothesis obtained by sign-flipping (one-sample test).
-
     Parameters
     ----------
-
     X : array-like of shape (n,p)
         numpy array of size [n,p], containing n observations of p variables
         (hypotheses)
     B : int
         number of sign-flippings to be performed (default=100)
-
     Returns
     -------
-
     pval0 : array-like of shape (B, p)
         A numpy array of size [B,p], whose rows are sorted increasingly.
         The entry i,j corresponds to p_{(j)}(g_i.X) with notation of [1]
         (section 4.5)_
-
     References
     ----------
-
     .. [1] Blanchard, G., Neuvial, P., & Roquain, E. (2020). Post hoc
         confidence bounds on false positives using reference families.
         Annals of Statistics, 48(3), 1281-1303.
     """
 
     np.random.seed(seed)
-
-    # Init
+    seeds = np.random.randint(np.iinfo(np.int32).max, size=B)
     n, p = X.shape
 
     # intialise p-values
-    pval0 = np.zeros((B, p))
+    pval0 = Parallel(n_jobs=-1)(delayed(
```

Comment:
Don't use `n_jobs=-1`, which can have a surprising behavior: it'll take all processors of the machine !
use an `n_job` argument, which defaults to 1 in some high-level function.